packages/kamino-sdk/src/Kamino.ts

```diff
@@ -129,23 +138,28 @@ export class Kamino {
    * @param cluster Name of the Solana cluster
    * @param connection Connection to the Solana cluster
    */
-  constructor(cluster: SolanaCluster, connection: Connection, globalConfig?: PublicKey) {
+  constructor(cluster: SolanaCluster, connection: Connection, programId: PublicKey, globalConfig: PublicKey) {
```

Comment:
Can we make the new arguments optional? If we add these then anyone using the sdk and updating it will have to fix every single reference and it will be painful to migrate. Would prefer if by default it fetches from config, if these two arguments arent present